spec/opencost-specv01.md

```diff
@@ -207,7 +206,19 @@ Workloads are defined as entities to which Asset Costs are committed. Some resou
   </tr>
 </table>
 
-
+The following workload cost aggregations are supported in a complete implementation in the OpenCost Spec: 
+
+* container
+* pod
+* deployment
+* statefulset
+* job
+* controller
```

Comment:
+1 on `controller name` will get that updated

Aggregating on `deployment` allows users to quickly see all active instances over the measured/projected window while also seeing costs that is not allocated to a deployment, e.g. jobs. This essentially allows users to skip a step when they are focused on measuring or optimizing one piece of their workloads at a time. Let me know if that makes sense. 

I think you've definitely highlighted a need to discuss filtering. That's potentially another standalone section of the spec which is actually already supported in the OpenCost implementation.

Reply:
Gotcha - thanks for the detail on aggregating on `deployment`. Definitely interesting, and perhaps achievable through filtering. Bottom line, I think some of this may just come down to UX implementation - as there could be a number of ways to get to what the user wants ;-)

Filtering may also be an aspect of UX; perhaps we can decide later if it makes sense to include in the Spec.

No additional comments from me!